tests/controller/models/v1/lab_test.py

```diff
@@ -198,8 +198,8 @@ def test_cpu_resource_validation() -> None:
         # ResourceQuantity.memory field must eventually get parsed into an int,
         # its validator can accept anything.
         LabResources(
-            requests=ResourceQuantity(cpu=1.1, memory="1Gi"),  # type: ignore[arg-type]
-            limits=ResourceQuantity(cpu=1, memory="1Gi"),  # type: ignore[arg-type]
+            requests=ResourceQuantity(cpu=1.1, memory="1Gi"),
+            limits=ResourceQuantity(cpu=1, memory="1Gi"),
```

Comment:
If this really isn't needed, then the comment should be removed as well, but I'm suspicious that something has gone wrong with type checking.

Reply:
These annotations are still needed on main with current mypy so I'm not sure what's going on here and why mypy thinks it's safe to drop them.

Reply:
I'm using mypy 1.19.0 (like main), and mypy is definitely writing stuff about that file to its cache, so it thinks it is checking it.

Reply:
If I add it back in:
1) the line is now more than 80 columns,
2)
```
nox > mypy --namespace-packages --explicit-package-bases noxfile.py tests
tests/controller/models/v1/lab_test.py:201: error: Unused "type: ignore" comment  [unused-ignore]
Found 1 error in 1 file (checked 53 source files)
nox > Command mypy --namespace-packages --explicit-package-bases noxfile.py tests failed with exit code 1
nox > Session typing failed.
```

Reply:
On `main`, doing what I think is what the nox command is doing:

```
adam@m3-wired:~/git/nublado$ cd controller; UV_PYTHON=None UV_PROJECT_ENVIRONMENT=$HOME/git/nublado/hub/.nox/typing mypy --namespace-packages --explicit-package-bases noxfile.py src tests
tests/models/v1/lab_test.py:201: error: Unused "type: ignore" comment  [unused-ignore]
tests/models/v1/lab_test.py:202: error: Unused "type: ignore" comment  [unused-ignore]
tests/models/v1/lab_test.py:214: error: Unused "type: ignore" comment  [unused-ignore]
tests/models/v1/lab_test.py:215: error: Unused "type: ignore" comment  [unused-ignore]
Found 4 errors in 1 file (checked 117 source files)
```

But `nox -s typing` is happy with them in there and unhappy without them.